src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -215,3 +208,77 @@ export default class WorkspaceContainer extends Component {
     ])
   }
 }
+
+
+export const wrapWorkspace = ({ breadcrumbs, activeTab, title }, content) => {
+  return class extends Component {
+    constructor(props) {
+      super(props)
+      this.state = StateHistory.get()
+      this.child = createRef()
+    }
+
+    render() {
+      const { workspaceError } = this.state
+
+      return workspaceError ? this.renderError() : this.renderSuccess()
+    }
+
+    renderSuccess() {
+      const { namespace, name } = this.props
+      const { workspace } = this.state
+
+      return h(WorkspaceContainer, {
+        namespace, name, activeTab, workspace,
+        title: _.isFunction(title) ? title(this.props) : title,
+        breadcrumbs: breadcrumbs(this.props),
+        refresh: async () => {
+          await this.refresh()
+          const child = this.child.current
+          const childRefresh = Object.getPrototypeOf(child).refresh
+          childRefresh && childRefresh.call(child)
```

Comment:
Could this just be `this.child.current.refresh()`? (with appropriate null guards)